app/client.py

```diff
@@ -49,7 +46,7 @@ def create_outside_temp(location_id, temp, time):
 
 def upload_video(location_id, hog_id, video_path, time):
     measurement = _create_measurement(location_id, "video", time, hog_id=hog_id)
-    logger.info(measurement)
+    print(measurement)
```

Comment:
Not sure why this changed from logger? Perhaps gabe didn't know where to look for the log file?

Reply:
This branch was made from master when it should have been made from the sftp branch, which doesn't have logging atm